cookbook/oraclevs.md

```diff
@@ -0,0 +1,273 @@
+# Oracle AI Vector Search with LangchainJS Integration
+## Introduction
+Oracle AI Vector Search enables semantic search on unstructured data while simultaneously providing relational search capabilities on business data, all within a unified system. This approach eliminates the need for a separate vector database, reducing data fragmentation and improving efficiency.
+
+By integrating Oracle AI Vector Search with Langchain, you can build a powerful pipeline for Retrieval Augmented Generation (RAG), leveraging Oracle's robust database features.
+
+Key Advantages of Oracle Database
+Oracle AI Vector Search is built on top of the Oracle Database, providing several key features:
+
+* Partitioning Support
+* Real Application Clusters (RAC) Scalability
+* Exadata Smart Scans
+* Geographically Distributed Shard Processing
+* Transactional Capabilities
+* Parallel SQL
+* Disaster Recovery
+* Advanced Security
+* Oracle Machine Learning
+* Oracle Graph Database
+* Oracle Spatial and Graph
+* Oracle Blockchain
+* JSON Support
+## Guide Overview
+This guide demonstrates how to integrate Oracle AI Vector Search with Langchain to create an end-to-end RAG pipeline. You'll learn how to:
+
+* Load documents from different sources using OracleDocLoader.
+* Summarize documents inside or outside the database using OracleSummary.
+* Generate embeddings either inside or outside the database using OracleEmbeddings.
+* Chunk documents based on specific needs using OracleTextSplitter.
+* Store, index, and query data using OracleVS.
+## Getting Started
+If you're new to Oracle Database, consider using the free Oracle 23 AI Database to get started.
```

Comment:
Use the correct product name.

Link to info that the user can follow to install / get access to a DB